netbout-web/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "netbout-web",
+    "version": "2.16.2",
```

Comment:
I took it from netbout-web/bower.json. Is there a reasonable value? Or can it be omitted?